printer/printer.lua

```diff
@@ -510,7 +520,12 @@ end
 
 
 function M.update(self, dt)
-	self.node_parent_pos = gui.get_position(self.node_parent)
+	-- Only update node_parent_pos if we're not currently shaking
+    if self.shake_time <= 0 then
+        self.node_parent_pos = gui.get_position(self.node_parent)
+    end
+
+	-- self.node_parent_pos = gui.get_position(self.node_parent)
```

Comment:
And here

Reply:
Seems as a valid point to add this fix! Thanks a lot

Let's make small adjustments?

This getters in update pos looks weird honestly, but it also how it was and I thought "ooh that's why"